tests/test_e2e_workflow.py

```
@@ -0,0 +1,169 @@
+    """Test input validation works correctly"""
+    print("Testing input validation...")
+
+    # Valid inputs
+    assert InputValidator.validate_pr_number(123) == 123
+    assert InputValidator.validate_repository("owner/repo") == ("owner", "repo")
+    assert InputValidator.validate_model_name("o4-mini") == "o4-mini"
+
+    # Invalid inputs should exit
+    try:
+        InputValidator.validate_pr_number(-1)
+        assert False, "Should have failed on negative PR number"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)